src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -152,6 +152,12 @@ private function computeSubresourceOperations(RouteCollection $routeCollection,
                 'collection' => $isCollection,
             ];
 
+            $visiting = "$resourceClass$subresource";
```

Comment:
Oops... The visited tracking is not specific enough.

For example, if I have:

Product -> isRelatedTo
and
Product -> isSimilarTo

It only works for the first of its type.